assets/javascripts/discourse/widgets/discourse-reactions-state-panel.js.es6

```diff
@@ -55,20 +55,23 @@ export default createWidget("discourse-reactions-state-panel", {
       return;
     }
 
+    const divContainer = attrs.state.postIds.includes(attrs.post.id) ? h(
+      "div.counters",
+      attrs.post.reactions.map(reaction =>
+        this.attach("discourse-reactions-state-panel-reaction", {
+          reaction,
+          users: attrs.state[reaction.id],
+          post: attrs.post,
+          isDisplayed: reaction.id === this.state.displayedReactionId
+        })
+      )
+    ) : h('div.center', h('div.spinner'));
+
     return [
       ,
       h(
         "div.container",
-        h(
-          "div.counters",
-          attrs.post.reactions.map(reaction =>
-            this.attach("discourse-reactions-state-panel-reaction", {
-              reaction,
-              post: attrs.post,
-              isDisplayed: reaction.id === this.state.displayedReactionId
-            })
-          )
-        )
+        divContainer
```

Comment:
I would also prefer a better name than divContainer, I don't really care about the html implementation here, what is this container about? what does it contain?